rust-lib/src/lib.rs

```diff
@@ -346,3 +356,61 @@ pub extern "C" fn transfer(
     )
     .into()
 }
+
+fn _accept(
+    runtime: &COpaqueStruct,
+    consignment_file: *mut c_char,
+    reveal_outpoints: *mut c_char,
+) -> Result<(), RequestError> {
+    let runtime = Runtime::from_opaque(runtime)?;
+
+    let c_consignment_file = unsafe { CStr::from_ptr(consignment_file) };
+    let consignment_file = PathBuf::from(c_consignment_file.to_str()?);
+    let consignment = Consignment::read_file(consignment_file)?;
+
+    let reveal_outpoints: Vec<bp::blind::OutpointReveal> =
+        serde_json::from_str(&ptr_to_string(reveal_outpoints)?)?;
+
+    info!(
+        "AcceptArgs {{ consignment: {:?}, reveal_outpoints: {:?} }}",
+        consignment, reveal_outpoints
+    );
```

Comment:
probably better not to print all consignment in a debug form (`{:?}`) in the info command.

```suggestion
    debug!(
        "Accepting consigment for outpoints {:?} }}",
        reveal_outpoints
    );
    trace!("Consignment data: {:?}", consignment);
```

Reply:
I agree that `info` is too much (I've used it because all other functions are using it, but probably all logging in the sdk needs to be reviewed).

But I think that we can keep `info` to show that we received a request (without logging all the details) and then use `debug` to log the details, I don't get why we should use `trace` in this case

Reply:
since you proposed different changes to logging (in this PR and in #22), both on existing code and in upcoming code, I'd open an issue where to discuss logging and come with a general solution

Reply:
issue #23 opened